config/routes.rb

```diff
@@ -5,6 +5,11 @@
   ActiveAdmin.routes(self)
   root to: "home#index"
 
+  as :user do
+    get 'users/edit' => 'devise/registrations#edit', as: 'edit_user_registration'
```

Comment:
I believe `devise_for :users` will generate all needed routes to manage registration related actions. why would we add custom routes?

Reply:
We disabled the registerable module in our user model, so we couldn't access this endpoint. That's why I added it manually